### PR TITLE
feat: [PL-26099]: ADO connector changes

### PR DESCRIPTION
**Pull request by sunnykesh-harness**
### Summary
Changes are:

- To remove account level support
- Clean up required input and validation for account level support
- Modify labels and placeholders to explicit texts for project level support

Supporting BE PR: https://github.com/harness/harness-core/pull/34876
PR env: [https://pr.harness.io/pl-26055-ado/ng/#/account/kmpySmUISimoRrJL6NL73w/settings/resources/connectors](https://pr.harness.io/pl-26055-ado/ng/#)

#### Screenshots

**Before changes**
Screenshot 2022-07-06 at 10 45 29 AM

**After changes:**
Screenshot 2022-07-06 at 10 08 32 AM

#### PR Checklist

<details>

- [ Done] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ InProgress] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by sunnykesh-harness** [comment]
dockerize

**Comment by sunnykesh-harness** [comment]
dockerize

**Comment by sunnykesh-harness** [comment]
retrigger coverage

File: src/modules/10-common/strings/strings.en.yaml [hunk 1]
```diff
@@ -350,14 +350,13 @@ git:
   gitLabUrlPlaceholderSSH: 'git@gitlab.com:account/'
   bitbucketUrlPlaceholder: 'https://www.bitbucket.com/account/'
   bitbucketPlaceholderSSH: 'git@bitbucket.com:account/'
-  azureReposUrlPlaceholder: 'https://dev.azure.com/<organisationName>'
+  azureReposUrlPlaceholder: 'https://dev.azure.com/<organisationName>/<projectName>'
   azureReposUrlPlaceholderRepoHttp: 'https://dev.azure.com/<organisationName>/<projectName>/<repositoryName>'
```
Comment (mayankVermaHarness) [inline src/modules/10-common/strings/strings.en.yaml:354]: This is a small legacy bug—Can we add `_git` here like this:

`https://dev.azure.com/<organisationName>/<projectName>/_git/<repositoryName>`


image

**Review by mayankVermaHarness — DISMISSED** [review]

File: src/modules/10-common/strings/strings.en.yaml [hunk 2]
```diff
@@ -350,14 +350,13 @@ git:
   gitLabUrlPlaceholderSSH: 'git@gitlab.com:account/'
   bitbucketUrlPlaceholder: 'https://www.bitbucket.com/account/'
   bitbucketPlaceholderSSH: 'git@bitbucket.com:account/'
-  azureReposUrlPlaceholder: 'https://dev.azure.com/<organisationName>'
+  azureReposUrlPlaceholder: 'https://dev.azure.com/<organisationName>/<projectName>'
```
Comment (reetika04) [inline src/modules/10-common/strings/strings.en.yaml:353]: organization?

**Review by reetika04 — APPROVED** [review]

**Review by vkbansal — APPROVED** [review]

**Pull request merged**